infrastructure/code/code_html.go

```diff
@@ -68,11 +108,91 @@ func getDetailsHtml(issue snyk.Issue) string {
 
 	dataFlowHtml := getDataFlowHtml(additionalData)
 
+	// Header
 	html := replaceVariableInHtml(detailsHtmlTemplate, "issueId", issue.ID)
 	html = replaceVariableInHtml(html, "issueTitle", additionalData.Title)
+	html = replaceVariableInHtml(html, "issueType", getIssueType(additionalData))
 	html = replaceVariableInHtml(html, "severityText", issue.Severity.String())
-	html = replaceVariableInHtml(html, "dataFlow", dataFlowHtml)
+	html = replaceVariableInHtml(html, "severityIcon", getSeverityIconSvg(issue))
+
+	html = replaceVariableInHtml(html, "issueOverview", additionalData.Message)
+	// Data flow
 	html = replaceVariableInHtml(html, "dataFlowCount", fmt.Sprintf("%d", len(additionalData.DataFlow)))
+	html = replaceVariableInHtml(html, "dataFlow", dataFlowHtml)
+
+	// External example fixes
+	html = replaceVariableInHtml(html, "repoCount", fmt.Sprintf("%d", additionalData.RepoDatasetSize))
+	html = replaceVariableInHtml(html, "exampleCount", fmt.Sprintf("%d", len(additionalData.ExampleCommitFixes)))
+	html = replaceVariableInHtml(html, "tabsNav", getTabsHtml(additionalData.ExampleCommitFixes))
 
 	return html
 }
+
+func getFileName(filePath string) string {
```

Comment:
This function is not used anywhere now other than the tests, do we still need it?

Reply:
I saw you addressed the file name with `filepath.Base` 🙌 Thank you!

I just ran `filepath.Base()` against the test cases and a few don't pass:


| Test Case                          | Input                             | Expected Output | Actual Output                    |
|------------------------------------|-----------------------------------|-----------------|----------------------------------|
| Windows_path                       | `C:\Users\johnDoe\project\file.txt` | `file.txt`     | `C:\Users\johnDoe\project\file.txt` |
| Trailing_slash                     | `/Users/johnDoe/project/`         | (empty)      | `project`                        |
| Empty_string                       | (empty)                           | (empty)      | `.`                              |
| Only_slashes                       | `/////`                           | (empty)      | `/`                              |
| Nested_directories_with_no_file    | `/a/b/c/d/e/`                     | (empty)      | `e`                              |
| Unix_root_directory                | `/`                               | (empty)      | `/`                              |

But I guess these are edge cases that we don't need to take into consideration, right?

I also tested `filepath.Base()` with `juice-shop` and it seems is working fine:

```html
<span 
  class="data-flow-filepath" 
  file-path="/Users/cata/git/juice-shop/test/api/quantityApiSpec.ts"
  start-line="227"
  end-line="227"
  start-character="8"
  end-character="17">quantityApiSpec.ts:228</span>
<span 
  class="data-flow-filepath" 
  file-path="/Users/cata/git/juice-shop/test/api/passwordApiSpec.ts"
  start-line="28"
  end-line="28"
  start-character="12"
  end-character="21">passwordApiSpec.ts:29</span>
<span 
  class="data-flow-filepath" 
  file-path="/Users/cata/git/juice-shop/test/api/deliveryApiSpec.ts"
  start-line="22"
  end-line="22"
  start-character="10"
  end-character="19">deliveryApiSpec.ts:23</span>

```

I'm deleting the function and its test 🧹

Reply:
Hmm these are good things to consider. I wonder how this behaves now for those edge cases. I wouldn't worry for now but I'll raise a ticket to worry about later. Thanks for calling these out!

Reply:
https://snyksec.atlassian.net/browse/IDE-231

Reply:
I think we'll need it once the dataflow navigation is added via javascript